src/Libraries/DynamoArduino/Arduino.csproj

```diff
@@ -102,4 +102,4 @@
   <Target Name="AfterBuild">
```

Comment:
Please do a `git checkout master src/Libraries/DynamoArduino/Arduino.csproj` do revert this file, it is unnecessarily polluting the PR.